deploy/crds/applicationmonitoring.integreatly.org_blackboxtargets_crd.yaml

```diff
@@ -0,0 +1,71 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: blackboxtargets.applicationmonitoring.integreatly.org
+spec:
+  group: applicationmonitoring.integreatly.org
+  names:
+    kind: BlackboxTarget
+    listKind: BlackboxTargetList
+    plural: blackboxtargets
+    singular: blackboxtarget
+  scope: ""
+  validation:
+    openAPIV3Schema:
+      description: BlackboxTarget is the Schema for the blackboxtargets API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          description: BlackboxTargetSpec defines the desired state of BlackboxTarget
+          properties:
+            blackboxTargets:
+              description: 'INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
+                Important: Run "operator-sdk generate k8s" to regenerate code after
+                modifying this file Add custom validation using kubebuilder tags:
+                https://book.kubebuilder.io/beyond_basics/generating_crd.html'
+              items:
+                description: 'BlackboxtargetStructure contains: A target (url, module
+                  and service name) to be probed by the'
+                properties:
+                  module:
+                    type: string
+                  service:
+                    type: string
+                  url:
+                    type: string
+                required:
+                - module
+                - service
+                - url
```

Comment:
Are these all required?

Reply:
Yes these are required when creating a blackboxtarget

Reply:
Yep, these are all required to create a blackboxtarget